_posts/2022/2022-01-24-enqueue-jobs-quickly-with-sidekiq-bulk.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Enqueue Jobs Quickly with Sidekiq’s Bulk Features"
+description: "From thousands of network calls, to one."
+layout: article
+category: ruby
+image:
+  base: '2022/enqueue-jobs-quickly-with-sidekiq-bulk'
+  alt: "Martial Arts on the beach"
+  credit: "Thao Le Hoang"
+  source: "https://unsplash.com/photos/Xl-ilWBKJNk"
+
+---
+
+We often have to enqueue lots of the same type of job at the same time. Perhaps a data migration for all of one kind of object? Or a data processing step that will run faster if it's parallelised?
+
+Our focus is often on the potentially large queues or the elapsed time it takes to churn through all the work. There is another part of this that we have to consider: enqueuing all that work in the first place.
```

Comment:
*...or the time it will take to churn through...But there's another part of this that we have to...

Reply:
```suggestion
Our focus is often on the potentially large queues or the time it will take to churn through all the work. But there’s another part of this that we have to consider: enqueuing all that work in the first place.
```